ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/TlsConfiguration.scala

```diff
@@ -51,6 +52,20 @@ final case class TlsConfiguration(
           .build
       )
     else None
+
+  /** This is a side-effecting method. It modifies JVM TLS properties according to the TLS configuration. */
+  def setJvmTlsProperties(): Unit =
+    if (enabled && revocationChecks) enableOCSP()
+
+  /**
+    * Enables certificate revocation checks with OCSP.
+    * See: https://tersesystems.com/blog/2014/03/22/fixing-certificate-revocation/
+    */
+  private def enableOCSP(): Unit = {
+    System.setProperty("com.sun.net.ssl.checkRevocation", "true")
```

Comment:
Is this Oracle/OpenJDK-specific? Does it need to work and will it work with e.g. an IBM JVM?

Reply:
That's a good question! I'll confirm that but I think IBM's JVM has different property for OCSP.

Reply:
It would make sense to inject this function so that we can switch between JVM specific settings/test this easily.